### PR TITLE
015/master

### PR DESCRIPTION
**Pull request by raq929**
Updates for cohort 015

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,9 +123,10 @@ $("#toctitle").on('click', function(event){
   console.log("event.target is ", event.target);
 });
 $("#toctitle").on('click', function(event){
-  console.log("event.target is ", event.type);
+  console.log("event.type is ", event.type);
```
Comment (RealWeeks) [inline README.md:126]: why event.target to event.type?

**Review by RealWeeks — CHANGES_REQUESTED** [review]
small change. Event.type?

**Comment by RealWeeks** [comment]
Everything else is great though

File: README.md (hunk 1 above)
Comment (raq929) [reply README.md:126]: Because we're displaying event.type! :)

Comment (raq929) [reply README.md:126]: event.target is displayed above.

Comment (RealWeeks) [reply README.md:126]: Makes sense, merged.

**Pull request merged**